MJxPrep.js

```diff
@@ -371,14 +384,13 @@ if (window.MJxPrep) {
       var funcEnd = bracketCloses === null ? bracketOpens : bracketCloses + 1
 
       var front = eqn.slice(0, funcStart)
-      var middle = '{:' + eqn.slice(funcStart, funcEnd) +  ':}'
-      eqn = front + middle + eqn.slice(funcEnd)
+      var middle = eqn.slice(funcStart, funcEnd)
+      eqn = front + wrapIfNotOverscript(null, middle) + eqn.slice(funcEnd)
```

Comment:
I think this is unnecessary, as nothing in the set contains parentheses, and this is only called for completed functions, right?

Reply:
`wrapFuncCalls` wraps incomplete function calls:

```js
wrapFuncCalls('cat(') // returns '{:cat:}(
wrapVariables('cat(') // returns 'cat('
```

Now, I'm not sure that wrapping incomplete function calls is actually necessary... I think when we encountered the issues with `x'/2` and `p(x)/2`, we adopted "wrap everything" as a philosophy, so I wrapped incomplete function calls, too. (Overscript issues has made me question that philosophy...)

Reply:
Ah, the point was more that the old code is equivalent to the new code, unless I'm missing something?

Reply:
I think that the test I added for `wrapFuncCalls` describes how we want it to behave for overscript symbols:

https://github.com/mitodl/mitx-grading-library/pull/170/files#diff-1d7052fc002008ab0813d9d4de53d3b7R191

The middle two cases for that test require this alteration to `wrapFuncCalls`.

Reply:
Ah, I see now. Very good! Thanks for the clarifications. For posterity, I'll add that `overline()` converts to `{:overline():}`, which is the desired behavior, and works in AsciiMath.